Code/Framework/AzTest/AzTest/aztest_files.cmake

```diff
@@ -7,15 +7,23 @@
 #
 
 set(FILES
```

Comment:
I've convinced myself that UnitTest and TestTypes can move into AzTest, but I'll need an explanation on why all test modules will need the other files proposed in this move.